components/gitpod-db/src/typeorm/entity/db-prebuild-info-entry.ts

```diff
@@ -31,4 +31,8 @@ export class DBPrebuildInfo {
         })(),
     })
     info: PrebuildInfo;
+
+    // This column triggers the db-sync deletion mechanism. It's not intended for public consumption.
+    @Column()
+    deleted?: boolean;
```

Comment:
The migration for this field is still missing, right?

Are you still working on this PR/should if be in "draft" mode? :thinking:

Reply:
This one actually has the field, just wasn't in the db model. Yes, for the other ones we need the migrations. I'm moving the PR to draft.

Reply:
> just wasn't in the db model

Oh really... :see_no_evil: